CHANGELOG.md

```diff
@@ -13,13 +13,15 @@
 * Update OpenTelemetry 1.19.0
 * Fixed ordering of JVM performance options [#8579](https://github.com/strimzi/strimzi-kafka-operator/issues/8579)
 * Log a warning when a KafkaTopic has no spec [#8465](https://github.com/strimzi/strimzi-kafka-operator/issues/8465)
+* Update Strimzi OAuth library to 0.13.0
```

Comment:
Is there any special thing, bug fixed or new feature that we should mention here as sub-topics?

Reply:
Maybe we could say:
```suggestion
* Updated Strimzi OAuth library to 0.13.0 with better support for KRaft, and some new features and bug fixes.
```

Reply:
Actually I can improve the Description in this PR ... describe the added config options, and add a link to PR:
```suggestion
* Updated Strimzi OAuth library to 0.13.0 [#8802](https://github.com/strimzi/strimzi-kafka-operator/issues/8802)
```

Reply:
I think it is better to use the first one rather than link to a PR.

Reply:
That said, `some new features and bug fixes` is more or less meaning less. So I think you do not need to add it. Also, is the KRaft support via the operator actually functional? Did you tried it?

Reply:
Agree, I would leave `some new features and bug fixes` out unless you list all of them :-)
I would leave just the the one related to KRaft.

Reply:
> Also, is the KRaft support via the operator actually functional? Did you tried it?
That's a good point. Let me try it.

Reply:
Works like a charm.

Reply:
Maybe you can in that case use something like this in the CHANGELOG?
```
* Updated Strimzi OAuth library to 0.13.0 with KRaft support
```

Reply:
Oh, I see ou already pushed it ... it looks fine. No need to use the suggestion above.